packages/botonic-core/src/router.js

```diff
@@ -47,7 +47,7 @@ export class Router {
       //get in childRoute if one has path ''
       let defaultAction
       if (
-        !routeParams.route.path &&
+        !(routeParams.route && routeParams.route.path) &&
```

Comment:
You're already checking `routeParams.route` in the next line.
Maybe you can add this check at the beginning instead of doing it in every `if`?

Reply:
sure, now simplified 💯